### PR TITLE
Added a haiku

### PR DESCRIPTION
**Pull request by aemolinas**

### CODE REVIEW

**Comment by aemolinas** [comment]
Please add hacktoberfest-accepted to this PR.

File: _haikus/microservice.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+layout: haiku
+author: Amar Molinas
+title: Microservice
+---
+
+# Microservice
```
Comment (ltagliaferri) [inline _haikus/microservice.md:7]:
```suggestion
```

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
The extra line is going to mess up our continuous verification, can you remove it, please?

**Comment by aemolinas** [comment]
I removed the extra spaces in the file. I hope I did this correctly. please let me know if I did not.

**Review by ltagliaferri — APPROVED** [review]
Thank you for this haiku and for contributing to open source 🎉

**Comment by aemolinas** [comment]
If possible I would like to use this as my first submission for Hactoberfest. Would you be willing to add the label hacktoberfest-accepted to this PR, please?

**Comment by ltagliaferri** [comment]
With the merge it counts as a hacktoberfest contribution

**Comment by aemolinas** [comment]
Thank you so much!

**Pull request merged**